### PR TITLE
adds ember bootstrap 4 RC

### PR DESCRIPTION
**Pull request by jaredgalanis**

### CODE REVIEW

File: source/2020-05-01-the-ember-times-issue-146.md [hunk 1]
```diff
@@ -59,6 +60,21 @@ Want to get started? Be sure to read the [official status update for the Ember W
 
 ---
 
+## [ember-bootstrap 4 Release Candidate 🛳](https://twitter.com/simonihmig/status/1255099221415510016)
+
+The folks at [kaliber5 @kaliber5](https://github.com/kaliber5) have put out a new 4.0.0-rc.0 release candidate for [ember-bootstrap](https://github.com/kaliber5/ember-bootstrap)! 🎉
```
Comment (ijlee2) [inline source/2020-05-01-the-ember-times-issue-146.md]: Optional: I think we can omit the specific version number to make the sentence sound more natural.
```suggestion
The folks at [kaliber5](https://github.com/kaliber5) have put out a new release candidate for [ember-bootstrap](https://github.com/kaliber5/ember-bootstrap)! 🎉
```

File: source/2020-05-01-the-ember-times-issue-146.md [hunk 2]
```diff
@@ -59,6 +60,21 @@ Want to get started? Be sure to read the [official status update for the Ember W
...
+
+This release candidate:
+
+- refactors to using tag-less components everywhere in preparation for a Glimmer rewrite.
+- drops Ember < 3.16, node 8 Package.
```
Comment (ijlee2) [inline source/2020-05-01-the-ember-times-issue-146.md]:
```suggestion
- drops Ember < 3.16, Node 8 Package.
```

File: source/2020-05-01-the-ember-times-issue-146.md [hunk 3]
```diff
@@ -59,6 +60,21 @@ Want to get started? Be sure to read the [official status update for the Ember W
...
+- drops deprecated arguments for HTML attributes
+- overhauls of the build setup, using embroider macros!!!
```
Comment (ijlee2) [inline source/2020-05-01-the-ember-times-issue-146.md]:
```suggestion
- overhauls the build setup, using embroider macros!!!
```

File: source/2020-05-01-the-ember-times-issue-146.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 title: The Ember Times - Issue No. 146
-author: Abhilash LR, Amy Lam, Jessica Jordan, Isaac Lee,  Chris Ng, the crowd
+author: Abhilash LR, Amy Lam, Jessica Jordan, Isaac Lee,  Chris Ng, Jared Galanis the crowd
```
Comment (ijlee2) [inline source/2020-05-01-the-ember-times-issue-146.md]: Unrelated to your PR. :)
```suggestion
author: Abhilash LR, Amy Lam, Jessica Jordan, Isaac Lee, Chris Ng, Jared Galanis the crowd
```

**Review by ijlee2 — APPROVED** [review]
Thanks for covering the news!

**Comment by amyrlam** [comment]
great suggestions, thanks @jaredgalanis @ijlee2 !

**Pull request merged**